gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/dsl/AppCdsSettings.kt

```diff
@@ -0,0 +1,181 @@
+package org.jetbrains.compose.desktop.application.dsl
+
```

Comment:
> Features - Desktop
> Implemented support for AppCDS

I would change it to Highlights - Desktop, and describe the feature as in my suggestion for KDoc.

Reply:
Worth to describe it more, for example "Implemented support for AppCDS that allows to significantly speed up application startup"

Reply:
Done.